### PR TITLE
Bug 1192726 - Convert forward locked radio controls to use web components

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1192726

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=126200e2935ca89bd82b9264278ce2193c32bd87

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83e2899896497dcc1882c91dbedd8ea226bf00c5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83e2899896497dcc1882c91dbedd8ea226bf00c5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a2c5b4772589a891f9a80b9e10cda2d141654728

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=98376ee12f33b15a0682101b06497168fd5b36a4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ce2e13bddac98f1903e3d51147073f80528e8a7a

File: apps/fl/js/pick.js [hunk 1]
```diff
@@ -100,23 +99,29 @@ function pickRingtone(activity) {
...
+      if (radio.checked) {
         selectedRingtone = ringtone;
         play(ringtone);
       }
-    };
+    });
 
-    container.appendChild(dom);
+    listItem.appendChild(radio);
+    container.appendChild(radio);
```
Comment (KevinGrandon) [inline apps/fl/js/pick.js]: Doh, this was the bug. Should be listItem here instead.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=63d5fee65f2e6469501e94a43cf85405b72fc822

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4a7de96e2c9179b28490211fc1856ae35b3fcb3e

**Pull request merged**